### PR TITLE
Reduces Cluwne combat effectiveness.

### PR DESCRIPTION
**Pull request by TheOneAndOnlyCreeperJoe**
"Cluwnes have a 50% chance of dropping an item instead of throwing it"
  "Cluwnes have a 40% chance of dropping their current item if they attack someone with it"
  "Cluwnes have a 40% chance of dropping a spray bottle when you spray with it"

say goodbye to the days of cluwnes actually fighting back

### CODE REVIEW

**Comment by trifornt-tinfornt** [comment]
endless cluwne nerfs wew lads.
I get that my point might not come across as valid given that I'm someone who _wants_ to be a Cluwne, but you have to take the Wiz spell into consideration here. Every time you nerf Cluwnes, you buff the spell. This would be kind of a **huge** buff for the spell as pretty much no cursed person can fight back AT ALL.
I'm fine with it as I like the challenge but I'm pretty sure it's an instawin Wiz spell now.

**Comment by TheOneAndOnlyCreeperJoe** [comment]
Range was nerfed anyway. If it grows to be a problem it can be nerfed again.

I am however never nerfing it down to a touch based attack, because it would basically mean that Disintegrate once again becomes strictly superior to Curse of the Cluwne.

**Comment by trifornt-tinfornt** [comment]
Problem is that you're basically turning it into a ranged Ei Nath, as it can travel three tiles away from you and the person who is cursed is not only valid but is totally useless and even more hopeless in efforts to stop you.
When it gets to that point I'd just say it'd be better to Ei Nath.

**Comment by TheOneAndOnlyCreeperJoe** [comment]
Ei Nath permanently takes you out of the round and is one of the most un-fun things to play against.

Here you can still play and if you die you'll get cloned as a non-cluwne (unless cluwne banned).

**Comment by Lawligagger** [comment]
fantastic

**Comment by TDcoolguy300** [comment]
Checked the code, i questioned myself and answered it multiple times, A-Ok from me

File: code/modules/mob/living/carbon/carbon.dm [hunk 1]
```diff
@@ -262,6 +262,10 @@
 		unEquip(item)
 	if(src.client)
 		src.client.screen -= item
+	if(src.disabilities && CLUWNEMUT && prob(50))
```
Comment (JohnGinnane) [inline code/modules/mob/living/carbon/carbon.dm:265]: inb4 "Carbonhell: sssssrrrrrccccccccc"

Note: This isn't a complaint

Comment (TDcoolguy300) [inline code/modules/mob/living/carbon/carbon.dm:265]: It appears only src can be properly used there, no other method unless you want to add a useless typecast

File: code/modules/reagents/reagent_containers/spray.dm [hunk 2]
```diff
@@ -18,6 +18,11 @@
 
 
 /obj/item/weapon/reagent_containers/spray/afterattack(atom/A as mob|obj, mob/user)
+	if(user.disabilities && CLUWNEMUT && prob(40))
```
Comment (JohnGinnane) [inline code/modules/reagents/reagent_containers/spray.dm:21]: I'm wondering, what if there's a (even smaller) chance you spray yourself with the bottle?

**Comment by JohnGinnane** [comment]
Code looks good!

**Comment by TheOneAndOnlyCreeperJoe** [comment]
oh my god why does it think there's conflicts there's nothing wrong.

**Comment by JohnGinnane** [comment]
Possibly with Crystal's recently merged dismemberment?

**Comment by Crystalwarrior** [comment]
if you have sourcetree solving conflicts is child's play............. :^)

**Comment by TheOneAndOnlyCreeperJoe** [comment]
Closing because I have to like, fucking fix conflicts and sourcetree is a little bitch (and Crystal's code is breaking the server)

**Pull request closed without merging**